sqlglot/dialects/snowflake.py

```diff
@@ -290,6 +290,7 @@ class Parser(parser.Parser):
             ),
             "NULLIFZERO": _build_if_from_nullifzero,
             "OBJECT_CONSTRUCT": _build_object_construct,
+            "OBJECT_INSERT": exp.ObjectInsert.from_arg_list,
```

Comment:
why is this needed if we have an ObjectInsert expression? shouldn't it auto parse?

Reply:
Right, will remove it.